KsApi/GraphIDBridging.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+public protocol GraphIDBridging {
```

Comment:
I've added this as a simple protocol that our models can conform to if they're able to return a GraphQL relay ID which is the concatenation of a model name, hyphen and the integer ID, base64-encoded. Essentially all the model needs to do is to conform to this protocol and return its model name, e.g. `"Project"`.